static/app/views/monitors/utils/crontabAsText.tsx

```diff
@@ -0,0 +1,23 @@
+import cronstrue from 'cronstrue';
+
+import {shouldUse24Hours} from 'sentry/utils/dates';
+
+/**
+ * Display a human readible label of a crontab expression
+ */
+export function crontabAsText(crontabInput: string | null): string | null {
+  if (!crontabInput) {
+    return null;
+  }
+  let parsedSchedule: string;
+  try {
+    parsedSchedule = cronstrue.toString(crontabInput, {
```

Comment:
```suggestion
    return cronstrue.toString(crontabInput, {
```
You can drop the tmp var